Anlab.Mvc/AnlabMvc.csproj

```diff
@@ -35,6 +35,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="2.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="2.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="2.0.2" />
```

Comment:
@srkirkland Should I delete this?

Reply:
@jSylvestre probably doesn't hurt anything since it's a VS specific thing, but I would if you don't care about it